Make some additional troubleshooting adjustments (#396)
home-assistant/companion.home-assistant: docs/notifications/basic.md

        title: Motion has been detected
```

> By default Text To Speech notifications use the music stream so they will bypass the ringer mode on the device given that volume is not at 0. You have the option of using `channel: alarm_stream` to have your notification spoken regardless of music volume.

Reviewer: ```suggestion
By default Text To Speech notifications use the music stream so they will bypass the ringer mode on the device as long as the device's volume is not set to 0. You have the option of using `channel: alarm_stream` to have your notification spoken regardless of music volume.
```
Maybe clearer? I assume this refers to the device volume.

Author: Yup this is clearer :)
